packages/cli/tsconfig.json

```diff
@@ -12,7 +12,8 @@
     "noEmitOnError": false,
     "skipLibCheck": true,
     "types": ["jest"],
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "lib": ["WebWorker"]
```

Comment:
Why is this needed ? For which functionality ?

Reply:
Because platform-client is using APIs not available in NodeJs ([Blob](https://developer.mozilla.org/en-US/docs/Web/API/Blob), https://developer.mozilla.org/en-US/docs/Web/API/File).
We only need the typing and not the actual implementation.

Reply:
Thanks to @louis-bompart for this one :P